http_example_test.go

```diff
@@ -150,7 +151,7 @@ func createToken(user string) (string, error) {
 		&jwt.StandardClaims{
 			// set the expire time
 			// see http://tools.ietf.org/html/draft-ietf-oauth-json-web-token-20#section-4.1.4
```

Comment:
While we are at: Changing this reference to https://datatracker.ietf.org/doc/html/rfc7519#section-4.1.4 would be appropriate as well. That is btw the source of the "wrong" implementation. When the library was initially started, RFC7519 did not exist - only its drafts. The referenced draft actually stated that `iat`, `exp` and so on must be a `IntDate`, which did not include the float part. It got then later changed to a `NumericDate` format in draft version 26, which - as you correctly pointed out - included also float representations.